ArduPlane/defines.h

```diff
@@ -135,7 +139,9 @@ enum log_messages {
     LOG_STATUS_MSG,
     LOG_OPTFLOW_MSG,
     LOG_QTUN_MSG,
-    LOG_PARAMTUNE_MSG
+    LOG_PARAMTUNE_MSG,
+    LOG_THERMAL_MSG,
+    LOG_VARIO_MSG
```

Comment:
Even for soaring, you aren't necessarily interested in both of these pieces of information at once. They are used separately in the code.